musicbot/config.py

```diff
@@ -355,6 +357,8 @@ class ConfigDefaults:
     legacy_skip = False
     leavenonowners = False
     usealias = True
+    disableembedfooter = True
```

Comment:
The default for this should be `False`

Reply:
since we're setting the default footer in config.py there's really no need for this option anymore so I just ended up removing this